docs/charter.md

```diff
@@ -147,58 +150,52 @@ This means that the products of our work should be considered _best effort_, and
+  - Introduce tooling for downstream verification of provenance, chain-of-trust and chain-of-custody
+- [Metadata and Software Bill of Materials](https://github.com/orgs/CPAN-Security/projects/1)
+  - Tooling for the creation and verification of SBOM objects commonly used to communicate the composition and pedigree of CPAN dependencies, even when these dependencies cross ecosystem boundraries.
+  - Update CPAN package metadata to enable end-users to comply with legislative obligations.
+- [Vulnerability Index](https://github.com/orgs/CPAN-Security/projects/10)
+  - Standardization and publishing of CPAN package vulnerabilities in relevant indexes (our own, CPANSA, CVE or whatever), to ensure that common (including third-party) tooling for Software Composition Analysis or vulnerability assessment work with CPAN dependencies.
+- [Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+  - Improve or create user-interfacing tooling used for analyzing dependencies for known vulnerabilities.
+- [Security Patch Tooling](https://github.com/orgs/CPAN-Security/projects/11)
+  - Develop tooling for publishing and applying third-party security patches to CPAN distributions that have non-responsive authors, to enable high-priority updates to CPAN packages.
+  - Explore opportunities for simplifying downstream fixes to reach upstream authors.
+- [Security Outreach & Information](https://github.com/orgs/CPAN-Security/projects/12)
+  - Establish a website and social media presence for outreach and information sharing, and keep these up to date.
+- [CPANSec Governance, Policy & Funding](https://github.com/orgs/CPAN-Security/projects/7)
+  - Establish procedures for responsible disclosure, communication and other related mechanisms, including documentation on how vulnerabilities and malicious code incidents are handled.
+  - Create a group charter and accountability procedures.
+  - Raise funds for CPAN Security Group projects.
 
 
 ### Tools
 
-- TBD
+- TODO
 
 
 ### Public contact points
 
-- TBD
-- https://security.metacpan.org/
-- ircs://irc.perl.org/#cpan-security
-- &lt;cpan-security@perl.org&gt; (mailing list)
-- Presence on Mastodon, Twitter, etc.
-- …
+- Main website: [https://security.metacpan.org/](https://security.metacpan.org/)
+- Chat (IRC): [ircs://ssl.irc.perl.org:7062/#cpan-security](ircs://ssl.irc.perl.org:7062/#cpan-security)
+- Fediverse: [@cpan_security@perl.social](https://perl.social/profile/cpan_security)
+- Email: &lt;cpan-security&#64;perl.org&gt;
 
 
 ### Public resources, feeds & channels
 
-- TBD
-- RSS/Atom feed with advisories
-- Blog under a well-known domain (e.g. security.metacpan.org)
-- Github group: https://github.com/CPAN-Security
-- …
+- [CPANSec News RSS feed](https://security.metacpan.org/feed.xml).
+- Github group: [https://github.com/CPAN-Security](https://github.com/CPAN-Security)
+- TODO: RSS/Atom feed with advisories
```

Comment:
same here. I'd remove this entirely until it is ready. The project itself is already listed above, there's no need for the placeholder.

Reply:
Done!